key/store.go

```diff
@@ -0,0 +1,222 @@
+package key
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/libp2p/go-libp2p-core/peerstore"
+	"github.com/qri-io/qri/config"
+	"github.com/theckman/go-flock"
+)
+
+// Store is an abstraction over a libp2p.KeyBook
+// In the future we may expand this interface to store symmetric encryption keys
+type Store interface {
+	peerstore.KeyBook
+
+	Owner() (peer.ID, crypto.PrivKey)
```

Comment:
I know I wrote this, but I think we should remove the `Owner` method, and the concept of storing "owner" state with the `key.Store` interface entirely. I'd rather see the current user cached in configuration / on an instance, and keep the key store as in-the-dark as possible.

If all this interface is doing is abstracting `keystone.KeyBook` and adding persistence, that's a great step IMHO.

Reply:
glad to see this, the owner provides little to no value here and is just extra stuff to carry around. It's not even persisted in the current context but loaded from config every time.